src/server/router.js

```diff
@@ -8,7 +8,7 @@ const router = (request, response)=> {
             handlers.publicHandler(request,response)
     }
     else if (request.url==='/getPost') {
-          handlers.getPostHandler(request,response)
+          handlers.getPostHandler(response)
     }
```

Comment:
This Handler needs to call the `validator` function and only respond after after the cookies is validated.